com.jetbrains.CLion.yml

```diff
@@ -0,0 +1,65 @@
+app-id: com.jetbrains.CLion
+command: clion-desktop
+runtime: org.freedesktop.Sdk
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+finish-args: 
+  - --device=all
+  - --filesystem=home
+  - --filesystem=xdg-run/keyring
+  - --share=ipc
+  - --share=network
+  - --socket=ssh-auth
+  - --socket=wayland
```

Comment:
Does it actually work natively on Wayland?

Reply:
I'm not sure but it matches what other Jetbrains apps on Flathub define in their manifest.

Reply:
I dropped wayland socket since Jetbrains/Java UI toolkit does not support it.